packages/itwinui-css/backstop/scenarios/footer.js

```diff
@@ -10,13 +10,15 @@ module.exports = [
   scenario('Type visible in viewport', {
     selectors: ['viewport'],
     viewports: [{ width: 800, height: 600 }],
+    hideSelectors: ['theme-button'],
```

Comment:
Did this not cause test image changes?

Reply:
This prevented test images from changing.  In this test, it scrolls down the page.  Previously the page settings button was `position: absolute` top right of the page and was above the fold.  Now with the page settings button being `position: fixed` top right, it will appear in our tests unless we hide the button.